rotkehlchen/db/filtering.py

```diff
@@ -332,6 +332,15 @@ def location(self) -> Optional[Location]:
         return self.location_filter.location
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBBooleanFilter(DBFilter):
```

Comment:
I don't think there's a need to create this filter.

You can add support for boolean to `DBEqualsFilter` since it does the exact same thing.
```python
@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
class DBEqualsFilter(DBFilter):
    """Filter a column by comparing its column to its value for equality."""
    column: str
    value: Union[str, bytes, int, bool]
    alias: Optional[str] = None

    def prepare(self) -> Tuple[List[str], List[Any]]:
        key_name = self.column
        if self.alias is not None:
            key_name = f'{self.alias}.{key_name}'
        return [f'{key_name}=?'], [self.value]
  ```